content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+Later on, the possibility to compile ReactOS with MSVC was added, and with it too, support for the WinDbg debugger. Kernel debugging with WinDbg works by having a debugging stub (KD64, this is a similar idea to GDB) on the target, that is capable of manipulating the state of the OS, and having the actual debugger offloaded to a host computer.
+
+Hermès is working on improving the ReactOS' kernel debugger (KDBG) with new features and miscellaneous fixes so that it makes easier for developers and testers alike to reliably debug ReactOS.
+The following improvements are:
+* Isolating terminal and input-related functionalities from the rest of KDBG ([#5188](https://github.com/reactos/reactos/pull/5188));
+* Fixed an input buffer reading bug in `KdbpReadCommand()` and `KdReceivePacket()` routines ([#4914](https://github.com/reactos/reactos/pull/4914));
+* Reworked the boot-phase initialization of KD and KDBG ([#4892](https://github.com/reactos/reactos/pull/4892));
+* Moved the code handling of Dmesg buffer from screen provider to KDBG provider ([#5143](https://github.com/reactos/reactos/pull/5143));
+
+Eventually the goal of this work is to have a terminal version of KDCOM and separate KDBG into a dedicated KDCOM-like extension DLL, such like `kdrosdbg.dll` (see ([#4600](https://github.com/reactos/reactos/pull/4600))).
+All of this effort is further coordinated with Timo Kreuzer.
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 is now capable of booting on an increasing number of hardware.
+With this new step we now are able to look at booting on more modern devices, the first step of this transition being the support of UEFI Class 3 boot (Systems without a BIOS fallback).
+Work has been going on since the beginning of the year to transition freeloader our default bootloader for ReactOS to support UEFI on ARM32, ARM64, x86 and of course primarily; AMD64.
+Developer Hermès BÉLUSCA - MAÏTO (HBelusca) has been developing a system for passing the UEFI framebuffer information in a fashion that could be supported even under older systems bringing even XP to UEFI!
+While developer Justin Miller (The_DarkFire_) has been developing the UEFI freeloader build.
+
+On top of just supporting booting ReactOS some other features are being built such as a bootmgfw compatible disguised build of FreeLdr so modern Windows 11 systems can bootstrap FreeLdr easily as well as EFI chainloading giving boot management capabilities to our favorite bootloader.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/uefi.png" src="/img/blogs/newsletter-103/uefi.png" caption="Running ReactOS on a Steam Deck" >}}
```

Comment:
Could add here two pics of FreeLoader on ARM devices (Lumia 950XL - ARM64, and HTC HD2 - ARM32), as well as the picture of ReactOS FreeLdr entry available in Windows Boot Manager. - I could attach these pics here later.